### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Microsoft.EntityFrameworkCore.SqlServer_ColorContrast :The color contrast ratio of the focus on the item in the top navigation is 2.061:1 which less than 3:1

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#8986]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468939

### Repro Steps

1. Launch the application URL: https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SqlServer/
2. Tab to the tap navigation
3. Run the Accessibility insights for web 
4. Check the color contrast for the selected items 

The color contrast ratio of the focus on the top selected item in the navigation is 2.061:1 which less than 3:1

Similar issue is observed in throughout the application for all the items in the navigation bar and controls that have a black focus outline with blue background

### Expected Behavior

The color contrast ratio for the focus must be 3:1 or greater
